flink-runtime/src/main/java/org/apache/flink/runtime/rpc/taskexecutor/SlotReport.java

```diff
@@ -18,21 +18,38 @@
 
 package org.apache.flink.runtime.rpc.taskexecutor;
 
+import org.apache.flink.runtime.clusterframework.types.ResourceID;
+
 import java.io.Serializable;
+import java.util.List;
 
 /**
  * A report about the current status of all slots of the TaskExecutor, describing
  * which slots are available and allocated, and what jobs (JobManagers) the allocated slots
  * have been allocated to.
  */
-public class SlotReport implements Serializable{
+public class SlotReport implements Serializable {
+
+	private static final long serialVersionUID = -3150175198722481689L;
+
+	/** the status of all slots of the TaskManager */
+	private final List<SlotStatus> slotsStatus;
 
-	private static final long serialVersionUID = 1L;
+	// resourceID identify the taskExecutor
```

Comment:
I know we're not overly consistent with that but field descriptions use normally the multi-line comment style `/** .... */`.